tests/flow/test_reduce.py

```diff
@@ -84,7 +84,7 @@ def test_missing_sections_reduction(self):
         try:
             self.graph.query(q).result_set
         except ResponseError as e:
-            self.env.assertIn("n not defined", str(e))
+            self.env.assertIn("Unknown function 'reduce'", str(e))
```

Comment:
The parser doesn't recognize the reduce pattern, so it is parsed as a function call and thus the error-message.
I think this can be a good sign to the user that he did something wrong, since as you said, `reduce` is not a function.